index.html

```diff
@@ -1,25 +1,345 @@
 <!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.7.2/css/all.css"
+      integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="icon" href="icons/youtube.png" />
+    <title>FULL FILM: McLaren Speedtail vs F35 Fighter Jet | Top Gear</title>
+  </head>
 
-<head>
-</head>
-
-<body>
-<div class="nav-bar">
-    <div class="logo">
-        <img src="https://upload.wikimedia.org/wikipedia/commons/9/90/Logo_of_YouTube_%282013-2015%29.svg">
-    </div>
-    <div class="search-bar">
-        <input type="Search">
-            <div class="search-button">
-                <button type="Search"></button>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <div class="left-header">
+          <span class="bar"> <i class="fa fa-bars"></i> </span>
+
+          <img
+            class="logo"
+            alt="youtube logo"
+            src="https://exame.com/wp-content/uploads/2017/08/new-youtube-logo-840x402.jpg?quality=70&strip=info"
+          />
+        </div>
+
+        <div class="center-header">
+          <input class="input" type="text" placeholder="Search" />
+          <button type="submit" class="search-button">
+            <i class="fas fa-search"></i>
+          </button>
+        </div>
+
+        <div class="right-header">
+          <i id="icon-video" class="fas fa-video" style="color: #606060;"></i>
+          <i id="icon-grid" class="fas fa-th" style="color: #606060;"></i>
+          <i
+            id="icon-setting"
+            class="fa fa-ellipsis-v"
+            style="color: #606060;"
+          ></i>
```

Comment:
### There should be no styles inside HTML files. Inline styling is a CSS bad practice. Please remove them.